### PR TITLE
Clean up import and import all

### PR DESCRIPTION
**Pull request by kootenpv**
Solves https://github.com/nostalgia-dev/nostalgia/issues/19

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -55,14 +55,22 @@
         "pyarrow",
         "tldextract",
         "diskcache",
```
Comment (jjc385) [inline setup.py:57]: This "diskcache" line can be deleted now (my bad for not noticing it was commented out lower in the file in my previous PR).

File: setup.py [hunk 2]
```diff
@@ -55,14 +55,22 @@
...
+        "requests_viewer",  # utils
         "tzlocal",
-        # "lxml",  # web_history
-        # "diskcache",  # web_history
-        # "auto_extract",  # web_history
-        # "ujson",  # web_history
-        # "google-api-python-client",  # offers
-        # "python-dotenv",  # offers
-        # "natura",  # offers
+        "lxml",  # web_history
```
Comment (jjc385) [inline setup.py]: I'd recommend against comments like this in the requirements.  They're nice right at this moment, but they're just waiting to become stale if another source uses a particular requirement.  If we want to to keep track of modular requirements, we'd be much better off recording them another way.  (E.g., we could have a json file with requirements listed for each data source, or a requirements.txt file in every data source's folder.)

File: docker-compose.yml [hunk 3]
```diff
@@ -1,7 +1,8 @@
 version: "3"
 services:
-  timeline:
+  nostalgia:
```
Comment (jjc385) [inline docker-compose.yml:3]: Possibly silly question (but good for review, I think) -- why this change?

File: nostalgia/unzip.py [hunk 4]
```diff
@@ -1,10 +0,0 @@
-import tqdm
-import sys
-import just
-import zipfile
-
-if __name__ == "__main__":
```
Comment (jjc385) [inline nostalgia/unzip.py:6]: I think it would be good practice to explain why this file was deleted (though I see the commit message says it's unnecessary), and maybe what it was once used for.

File: docker-compose.yml (hunk 3 above)
Comment (kootenpv) [reply docker-compose.yml:3]: it was copied from the timeline repo, but in core it should be named nostalgia :)

File: nostalgia/unzip.py (hunk 4 above)
Comment (kootenpv) [reply nostalgia/unzip.py:6]: It is unused simply. In favor of this I believe: https://github.com/nostalgia-dev/nostalgia/blob/master/nostalgia/extracter.py It's a generic way to unzip files into the right folder (using `ingest_settings`, see e.g. https://github.com/nostalgia-dev/nostalgia/tree/master/nostalgia/sources/google/__init__.py)

File: setup.py (hunk 2 above)
Comment (kootenpv) [reply setup.py]: Yea - that is the proper way in the future. You are totally right they will be stale so it is kinda useless!

File: setup.py (hunk 1 above)
Comment (kootenpv) [reply setup.py:57]: ah I see it's duplicate!

File: nostalgia/unzip.py (hunk 4 above)
Comment (jjc385) [reply nostalgia/unzip.py:6]: Thanks -- I'm glad I asked.

**Comment by kootenpv** [comment]
So basically I have processed all your comments @jjc385 Looking for one more...

**Review by jjc385 — APPROVED** [review]

**Review by NickolayVasilishin — APPROVED** [review]
